org-cyf/content/guides/employability/cvs/applicant-tracking-system/index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Applicant Tracking systems
+description: Make sure your CV stands out from the crowd
+emoji: 🖲️
+weight: 4
+---
+
+Many employers (especially larger companies) use **A**pplicant **T**racking **S**ystems (ATSs) to manage the huge amount of candidates that apply for roles online. An ATS is a system that scans resumes and matches them against criteria defined by the employer. These systems automatically filter out candidates before applications are seen by recruiters/hiring managers often removing as much as 75% of the candidates before a human has seen any of the CVs. **These systems must recognise your CV as strong to get it in front of a recruiter**.
+
+Practice preparing your CV to be successful for each application rather than sending out the same CV to every company. As well as understanding the principles of ATS there are some free tools you can use to practice and improve your CV.
```

Comment:
```suggestion
Practice preparing your CV to be successful for each application rather than sending out the same CV to every company. As well as understanding the principles of ATS, there are some free tools you can use to practice and improve your CV.
```